docs/courses/interacting_with_workflows/python.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Interacting with Workflows with Python"
+sidebar_position: 4
```

Comment:
Recommend giving this sidebar position 3 and the TypeScript course sidebar position 4 so that they'll be in alphabetical order